docs/modules/ROOT/pages/relay.adoc

```diff
@@ -160,6 +160,228 @@ All private keys are stored in the AWS Key Management Service. Keys are generate
 
 As for API secrets, these are only kept in memory during creation when they are sent to the client. After that, they are hashed and stored securely in AWS Cognito, which is used behind the scenes for authenticating Relayer requests. This makes API keys easy to rotate while preserving the same private key on the KMS.
 
+[[relay-api]]
+== Relay API
+Relay API exposes two endpoints `txs` and `sign`. The `txs` endpoint allows to send transactions to Ethereum blockchain and query their status. The `sign` endpoint allows to sign arbitrary data with relayer's private key.
+
+=== Getting JWT Token
+JWT Token is requred to make requests to Relay API. For security reasons authentication uses https://en.wikipedia.org/wiki/Secure_Remote_Password_protocol[SRP protocol] and it is not possbile to retrive a JWT Token via one HTTP request. What is why it is advised to use https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-integrate-apps.html[Amazon Cognito User Pool SDK] to retreive a JWT token.
+
+Here is an example of getting a JWT Token in nodejs using `amazon-cognito-identity-js`. The Username/Password pair is your ApiKey/ApiSecret pair. PoolId and PoolClientId are always the same.
```

Comment:
I'd remove this example and replace it with a link to the Cognito documentation on how to get a token, and maybe a very short snippet on how to do it with the AWS CLI (see [here](https://gist.github.com/Can-Sahin/377fee56a53a0a31bf9e5ce7f9847ac2)). I don't think it makes sense to provide a js snippet when the user could just use our own client if they are in js, and it also distracts a lot from the main reference.

Reply:
AWS CLI would not work for `USER_SRP_AUTH` because it requires to complete a challenge. I agree with `js` example being unnecessary.